notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Changed it.